source/_integrations/hassio.markdown

```diff
@@ -141,7 +141,7 @@ Create a full backup.
 
 | Service Data Attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `name` | yes | Name of the backup file. Default is current date and time
+| `name` | yes | Name of the backup file. Default is the current date and time in the user's local time
```

Comment:
It's the time zone configured in the general settings, I'm not sure how to best explain that.
![image](https://github.com/home-assistant/home-assistant.io/assets/14281572/f0fbada9-d052-4ba7-a386-781e594bad41)

Reply:
Something like `By default, the current date and time are used in your local time, which you have set in your general settings.`

Reply:
```suggestion
| `name` | yes | By default, the current date and time are used in your local time, which you have set in your general settings.
```